src/docfx/build/Build.cs

```diff
@@ -29,21 +30,36 @@ private static List<Document> GlobFiles(Context context, Docset docset)
                            .ToList();
         }
 
-        private static Task BuildFiles(Context context, List<Document> files)
+        private static Task BuildFiles(Context context, List<Document> files, TableOfContentsMap tocMap)
         {
-            return ParallelUtility.ForEach(files, file => BuildOneFile(context, file));
+            return ParallelUtility.ForEach(files, file => BuildOneFile(context, file, tocMap));
+        }
+
+        private static async Task<TableOfContentsMap> BuildTocMap(Context context, List<Document> files)
```

Comment:
move it to `BuildTableOfContents.cs`

Reply:
OK, make sense

---
In reply to: [184584857](https://github.com/dotnet/docfx/pull/2696#discussion_r184584857) [](ancestors = 184584857)